### PR TITLE
MTP-1295 move to sentry.io

### PR DESCRIPTION
**Pull request by Staberinde**

### CODE REVIEW

File: requirements/docker.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@
 
 money-to-prisoners-common[monitoring]>=9.18,<9.19
 
+sentry-sdk==0.16.5
```
Comment (ushkarev) [inline requirements/docker.txt:6]: is this here temporarily before mtp-common is updated?

File: README.md [hunk 2]
```diff
@@ -60,3 +60,15 @@ NB: you should pull updates before pushing to merge correctly.
 ## Deploying
 
 This is handled by [money-to-prisoners-deploy](https://github.com/ministryofjustice/money-to-prisoners-deploy/).
+
+## Documentation
+
+We have both swagger and redoc.io integration with this API. This is disabled on production, but can be found on the test environment:
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply README.md:66]: as for localhost URLs, it's deffo a good idea to include them

File: requirements/docker.txt (hunk 1 above)
Comment (Staberinde) [reply requirements/docker.txt:6]: Yup

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Staberinde) [reply README.md:66]: Hence why I didn't want to include the URLs. How about if I kept the localhost section and said something like `Similar pages are available on the test server`?

[the remark this answers is not in the document]
Comment (ushkarev) [reply README.md:66]: yeah i think that'd great

**Review by ushkarev — APPROVED** [review]

**Review by ushkarev — APPROVED** [review]

**Pull request merged**